tests/integration/evaluation/test_multi_table.py

```diff
@@ -47,12 +47,8 @@ def test_evaluation():
     score = evaluate_quality(data, samples, metadata).get_score()
     assert score == 0.9566297110928815
 
-    diagnostic = run_diagnostic(data, samples, metadata).get_results()
-    assert diagnostic == {
-        'DANGER': ['More than 50% of the synthetic rows are copies of the real data'],
-        'SUCCESS': [
-            'The synthetic data covers over 90% of the numerical ranges present in the real data',
-            'The synthetic data follows over 90% of the min/max boundaries set by the real data'
-        ],
-        'WARNING': []
-    }
+    report = run_diagnostic(data, samples, metadata)
+    assert report._overall_score == 1
+    assert report._properties['Data Validity']._compute_average() == 1
+    assert report._properties['Data Structure']._compute_average() == 1
+    assert report._properties['Relationship Validity']._compute_average() == 1
```

Comment:
can we use the public methods instead in case the internal stuff ever changes? Also no need to rerun the compute averages
```python
assert report.get_score == 1
pd.testing.assert_frame_equal(report.get_properties(), ...)
```